### PR TITLE
enabling kitchen, stove on workers

### PR DESCRIPTION
**Pull request by JoeNunnelley**
### Description

add the kitchen / stove / chef toolchains to workers for use in building cookbooks.

### Issues Resolved

allows us to move the cookbook build jobs off the server to a worker.

### Check List

- [ ] New functionality includes tests
- [ ] All tests pass
- [ ] CHANGELOG has been updated
- [ ] Version has been bumped in metadata
- [ ] Commits have been squashed, as appropriate
- [ ] Commits include descriptive messages, subjects written in the imperative

### CODE REVIEW

**Comment by JoeNunnelley** [comment]
retest this please

File: attributes/default.rb [hunk 1]
```diff
@@ -25,6 +25,9 @@
   sv['default-chef'] = '12.22.5'
 end
 
+default['chef']['workstation_version'] = '0.2.48'
```
Comment (RoboticCheese) [inline attributes/default.rb]: We should be able to safely go as far as 0.3.2, apparently the last release before they switched to the EULA in 0.4.1.

File: recipes/_chef.rb [hunk 2]
```diff
@@ -1,10 +1,45 @@
...
+# frozen_string_literal: true
+
+#
+# Cookbook Name:: socrata-jenkins
+# Recipe:: _chef
+#
+# Copyright:: 2014, Socrata, Inc.
+#
+# All rights reserved. Do not redistribute.
+include_recipe 'apt-chef'
```
Comment (RoboticCheese) [inline recipes/_chef.rb]: If we're not installing anything from their APT repo then the new dep on this cookbook is unnecessary.

File: attributes/default.rb (hunk 1 above)
Comment (JoeNunnelley) [reply attributes/default.rb]: done

File: recipes/_chef.rb (hunk 2 above)
Comment (JoeNunnelley) [reply recipes/_chef.rb]: ok

**Pull request merged**